app/views/HomeView.coffee

```diff
@@ -55,34 +46,43 @@ module.exports = class HomeView extends RootView
     @isTeacherWithDemo = @trialRequest and @trialRequest.get('status') in ['approved', 'submitted']
     super()
 
-  onClickLearnMoreLink: ->
-    window.tracker?.trackEvent 'Homepage Click Learn More', category: 'Homepage', []
-    @scrollToLink('#classroom-in-box-container')
-
   onClickRequestQuote: (e) ->
     @playSound 'menu-button-click'
     e.preventDefault()
     e.stopImmediatePropagation()
-    window.tracker?.trackEvent $(e.target).data('event-action'), category: 'Homepage', []
+    @homePageEvent($(e.target).data('event-action'))
     if me.isTeacher()
       application.router.navigate '/teachers/update-account', trigger: true
     else
       application.router.navigate '/teachers/quote', trigger: true
 
   onClickSetupClass: (e) ->
-    window.tracker?.trackEvent $(e.target).data('event-action'), category: 'Homepage', []
+    @homePageEvent($(e.target).data('event-action'))
     application.router.navigate("/teachers/classes", { trigger: true })
 
   onClickStudentButton: (e) ->
-    window.tracker?.trackEvent $(e.target).data('event-action'), category: 'Homepage', []
+    @homePageEvent($(e.target).data('event-action'))
     @openModalView(new CreateAccountModal({startOnPath: 'student'}))
 
   onClickTeacherButton: (e) ->
-    window.tracker?.trackEvent $(e.target).data('event-action'), category: 'Homepage', []
+    @homePageEvent($(e.target).data('event-action'))
     @openModalView(new CreateAccountModal({startOnPath: 'teacher'}))
 
   onClickTrackEvent: (e) ->
-    window.tracker?.trackEvent $(e.target).data('event-action'), category: 'Homepage', []
+    @homePageEvent($(e.target).data('event-action'))
+
+  # Provides a uniform interface for collecting information from the homepage.
+  # Always provides the category Homepage and includes the user role.
+  homePageEvent: (action, extraproperties={}, includeIntegrations=[]) ->
+    defaults =
+      category: 'Homepage'
+      user: me.get('role') || (me.isAnonymous() && "anonymous") || "homeuser"
+    properties = _.merge(defaults, extraproperties)
+    console.log(action, properties)
```

Comment:
Do we need this?
If yes, we can change it to console.debug and some more text for context.

Reply:
We don't need it. Great pickup. Removed